tools/cli/admin_db_clean_command.go

```diff
@@ -131,15 +132,19 @@ func fixExecution(
 	invariants []executions.InvariantFactory,
 	execution *store.ScanOutputEntity,
 ) (invariant.ManagerFixResult, error) {
-	execManager, err := getDeps(c).initializeExecutionStore(c, execution.Execution.(entity.Entity).GetShardID())
-	defer execManager.Close()
+	execManager, err := getDeps(c).initializeExecutionManager(c, execution.Execution.(entity.Entity).GetShardID())
+	if execManager != nil {
+		defer execManager.Close()
+	}
 	if err != nil {
-		return invariant.ManagerFixResult{}, fmt.Errorf("Error in fix execution: %w", err)
+		return invariant.ManagerFixResult{}, err
```

Comment:
Why are errors not wrapped?

Reply:
because it is wrapped in the caller side in `		if err != nil {
			return commoncli.Problem("Error in fix execution: ", err)
		}
`